tests/BootstrapperTest.php

```diff
@@ -184,30 +189,38 @@ public function filesystem_data_is_separated()
         $this->assertFalse(Storage::disk('public')->exists('foo'));
         $this->assertFalse(Storage::disk('public')->exists('abc'));
 
+        $expected_storage_path = $old_storage_path . '/tenant' . tenant('id'); // /tenant = suffix base
+
+        // Check that disk prefixes respect the root_override logic
+        $this->assertSame($expected_storage_path . '/app/', $this->getDiskPrefix('local'));
+        $this->assertSame($expected_storage_path . '/app/public/', $this->getDiskPrefix('public'));
+        $this->assertSame('tenant' . tenant('id') . '/', ltrim($this->getDiskPrefix('s3'), '/'));
```

Comment:
I'm using `ltrim()` in the tests here, to make the strings identical, but per what @CodeSkills suggested in https://github.com/archtechx/tenancy/pull/800#issuecomment-1038433316 there might be a problem with the prefixing? So if this issue happens in the app and not just tests, the `/` prefix adjustment should be made in the bootstrapper rather than tests.

Reply:
(For context, the tests don't use s3 at all. They just make some general config assertions.)

Reply:
@CodeSkills could you comment on the `/` prefix changes? It'd help a lot here